.github/workflows/entrypoint.yml

```diff
@@ -0,0 +1,29 @@
+name: Build doc
+
+on:
+  push:
+    paths-ignore:
+      - ".github/**"
+      - "**.md"
```

Comment:
Not sure this exception is relevant in this context, we might have some day markdown files in the documentation

Reply:
Agreed, so let's keep `.github/**`?